deploy-cloud-functions/action.yaml

```diff
@@ -0,0 +1,106 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Cloud Functions Deploy
+description: |-
+  Cloud Functions is a scalable Functions-as-a-Service (FaaS) to run your 
+  code with zero server management. Use this action to deploy your function or
+  update an existing Cloud Function.
+author: GoogleCloudPlatform
+
+inputs:
+  credentials:
+    description: |-
+      Optional service account key to use for authentication to GCS. This should be the JSON
+      formatted private key which can be exported from the Cloud Console. The
+      value can be raw or base64-encoded.
+    required: false
+    
+  name:
+    description: |-
+      Name of the Cloud Function.
+    required: true
+
+  description:
+    description: |-
+      Description for the CF.
```

Comment:
nit: consistency between Cloud Function and CF

Reply:
Moved user facing content to `Cloud Function`